clients/admin-ui/src/features/taxonomy/components/TaxonomyTreeNode.tsx

```diff
@@ -0,0 +1,139 @@
+  },
+  "taxonomyTreeNode"
+>;
+
+const TaxonomyTreeNode = ({
+  data,
+  positionAbsoluteX,
+  positionAbsoluteY,
+}: NodeProps<TaxonomyTreeNodeType>) => {
+  const { onMouseEnter, onMouseLeave, getNodeHoverStatus } = useContext(
+    TaxonomyTreeHoverContext,
+  );
+  const {
+    taxonomyItem,
+    onAddButtonClick,
+    onTaxonomyItemClick,
+    label,
+    hasChildren,
+    isLastCreatedItem,
+    resetLastCreatedItemKey,
+  } = data;
+
+  const { centerScreenOnNode } = useCenterScreenOnNode({
+    positionAbsoluteX,
+    positionAbsoluteY,
+    nodeWidth: 200,
+  });
+
+  useEffect(() => {
+    const centerScreenAndSimulateHover = async () => {
+      await centerScreenOnNode();
+      onMouseEnter(taxonomyItem?.fides_key!);
+    };
+
+    if (isLastCreatedItem) {
+      centerScreenAndSimulateHover();
+      resetLastCreatedItemKey();
+    }
+  }, [
+    isLastCreatedItem,
+    onMouseEnter,
+    taxonomyItem?.fides_key,
+    resetLastCreatedItemKey,
+    centerScreenOnNode,
+  ]);
+
+  const nodeHoverStatus = getNodeHoverStatus(taxonomyItem?.fides_key!);
+  const getNodeHoverStatusClass = useCallback(() => {
+    switch (nodeHoverStatus) {
+      case TreeNodeHoverStatus.ACTIVE_HOVER:
+        return styles["button-hover"];
+      case TreeNodeHoverStatus.PARENT_OF_HOVER:
+        return styles["button-hover-parent"];
+      case TreeNodeHoverStatus.INACTIVE:
+        return styles["button-inactive"];
+      case TreeNodeHoverStatus.DEFAULT:
+      case TreeNodeHoverStatus.CHILD_OF_HOVER:
+      case TreeNodeHoverStatus.SIBLING_OF_HOVER:
+        return styles["button-default"];
+      default:
+        return "";
+    }
+  }, [nodeHoverStatus]);
+
+  const isRootNode = taxonomyItem?.fides_key === TAXONOMY_ROOT_NODE_ID;
+
+  return (
+    <div
+      className={styles.container}
+      onMouseEnter={() => onMouseEnter(taxonomyItem?.fides_key!)}
+      onMouseLeave={() => onMouseLeave(taxonomyItem?.fides_key!)}
+      data-testid={`taxonomy-node-${taxonomyItem?.fides_key}`}
+    >
+      <AntButton
+        className={`${styles.button} ${getNodeHoverStatusClass()}`}
+        onClick={() => onTaxonomyItemClick?.(taxonomyItem!)}
+        disabled={!onTaxonomyItemClick}
+        type="text"
+      >
+        <AntTypography.Text ellipsis style={{ color: "inherit" }}>
+          {taxonomyItem?.active === false ? "(disabled) " : ""}
+          {label}
+        </AntTypography.Text>
+      </AntButton>
+
+      {!isRootNode && (
+        <TaxonomyTreeNodeHandle
+          type="target"
+          inactive={nodeHoverStatus === TreeNodeHoverStatus.INACTIVE}
+        />
+      )}
+      {hasChildren && (
+        <TaxonomyTreeNodeHandle
+          type="source"
+          inactive={nodeHoverStatus === TreeNodeHoverStatus.INACTIVE}
+        />
+      )}
+
+      <div
+        className={`
+          ${styles["add-button-container"]}
+          ${nodeHoverStatus === TreeNodeHoverStatus.ACTIVE_HOVER ? styles["add-button-container--visible"] : ""}`}
```

Comment:
Yes! Really great suggestion. I've moved this hidden/visible styling to the button instead of the container, and I added a :focus pseudoselector that makes it visible too. I've tested it and the button appears when you're using keyboard controls. Also added an aria-label for the button.